src/generated/api.d.ts

```diff
@@ -0,0 +1,239 @@
+/** Auto generated file from Schema Hash (8fd5beb9fb590a6cee9b0dda6c69498f8b4dc2eff9ac01374ae4ee7f635db665). DO NOT MODIFY **/
```

Comment:
The code depends on the generated files. Wouldn't that mean if you cloned the project, you would have to generate those files again, and they could possibly be different, creating breaking changes?

Reply:
It does, however the way we have the project structured is as soon as you do a pnpm dev or pnpm build it downloads the very latest schema definition anyways. We did this as a company decision because we found that more often than not developers would work off of old api methods or models than having the very latest version.

Our backend code we commit the generated files because we don't generate them on start and more times than not when you are working on the backend you are working on the schema so they stay in sync a lot easier than a disconnected front end.

So while we can commit these to code the if you look at the package.json for the "dev" script the first line will overwrite your generated folder as soon as you run it. Thus we have never comitted our generated folder on the front end.

Reply:
That's interesting. I can see how not committing them could be of benefit to the company. I like the concept of auto generating the files from the latest schema when you run the dev script.

For this repository, I don't think that happens though. I only see `vite dev`